edx_sga/__init__.py

```diff
@@ -2,5 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = '0.11.1'
-
+__version__ = '0.10.0'
```

Comment:
Can you remove this version number change? We prefer to handle this with our release bot.

Reply:
done